test/util/bspNet/helpers.ts

```diff
@@ -134,6 +135,10 @@ export const closeSimpleBspNet = async (verbose = false) => {
     container.Names.some((name) => name.includes("toxiproxy"))
   );
 
+  const postgresContainer = allContainers.find((container) =>
+    container.Names.some((name) => name.includes("docker-sh-postgres-1"))
+  );
```

Comment:
Not really needed in the BSP net, but yes in the full net

Reply:
The title is a bit of a misnomer as it's for fullnet too. They share the same cleanup function